Missing attributes on asyncio.LimitOverrunError and asyncio.IncompleteReadError

Sample code:
```python
import asyncio
import typing

def f(exc: asyncio.LimitOverrunError) -> None:
    print(exc.consumed)

def g(exc: asyncio.IncompleteReadError) -> None:
    print(exc.expected)
    print(exc.partial)
```

Mypy output:
```
demo.py:6: error: "LimitOverrunError" has no attribute "consumed"
demo.py:10: error: "IncompleteReadError" has no attribute "expected"
demo.py:11: error: "IncompleteReadError" has no attribute "partial"
```

I'm using Python 3.5.2, Mypy 0.521.

The types should be
- LimitOverrunError.consumed: int
- IncompleteReadError.expected: Optional[int]
- IncompleteReadError.partial: bytes

--- a/stdlib/3.4/asyncio/streams.pyi
+++ b/stdlib/3.4/asyncio/streams.pyi
@@ -12,9 +12,12 @@ _ClientConnectedCallback = Callable[[StreamReader, StreamWriter], Optional[Await
 __all__: List[str]
 
 class IncompleteReadError(EOFError):
-    def __init__(self, partial: str, expected: int) -> None: ...
+    expected = ...  # type: Optional[int]
+    partial = ...  # type: bytes
+    def __init__(self, partial: bytes, expected: Optional[int]) -> None: ...
 
 class LimitOverrunError(Exception):
+    consumed = ...  # type: int
     def __init__(self, message: str, consumed: int) -> None: ...
 
 @coroutines.coroutine

